File: Array/00-array.c
#include <stdio.h>

int main()
{
    int meu_array[5] = {15, 78, 12, 36, 5};
    int menorValor = meu_array[0];
    int tamanhoArray = sizeof(meu_array)/sizeof(meu_array[0]);

    for (int i=0; i < tamanhoArray; i++)
    {
        if (menorValor > meu_array[i])
        {
            menorValor = meu_array[i];
        }
    }

    printf("O menor valor do array é %d", menorValor);

    return 0;
}

File: Array/00-array.py
meu_array = [50, 17, 2, 38, 4, 57]

menorValor = meu_array[0]

for i in meu_array:
    if menorValor > i:
        menorValor = i

print(f"O menor valor do array é {menorValor}.")
